src/runtime/hexagon_dma.cpp

```diff
@@ -710,7 +710,7 @@ WEAK int halide_hexagon_dma_power_mode_voting(void *user_context, halide_hexagon
         val = ~PW_SVS;
         break;
     default:
-        error(user_context) << "Hexagon: halide_hexagon_dma_power_voting power mode (", cornercase, ") not found\n";
+        error(user_context) << "Hexagon: halide_hexagon_dma_power_voting power mode (" << cornercase << ") not found\n";
```

Comment:
Nice catch. How did you find it?

Reply:
Good question. I only realize now that you'd be very interested in catching this automatically. One of the compilers I used over the last weeks was complaining about it. Could have been clamg on macOS.